test/integration/template_selection_test.rb

```diff
@@ -4,167 +4,105 @@ class TemplateSelectionTest < ActionDispatch::IntegrationTest
   include Devise::Test::IntegrationHelpers
 
   setup do
-    scaffold_template
-    @template = Template.default
-
-    @researcher = User.last
-    scaffold_org_admin(@template.org)
-
-    @funder = Org.find_by(org_type: 2)
-    @funder_template = @funder.templates.where(published: true).first #Template.create(title: 'Funder template', org: @funder, archived: false)
-    # Template can't be published on creation so do it afterward
-    @funder_template.published = true
-    @funder_template.visibility = Template.visibilities[:publicly_visible]
-    @funder_template.save
-
-    @org = @researcher.org
-    @org_template = Template.create(title: 'Org template', org: @org, archived: false)
-    # Template can't be published on creation so do it afterward
-    @org_template.published = true
-    @org_template.visibility = Template.visibilities[:organisationally_visible]
-    @org_template.save
+    # Need to clear the tables until we get seed.rb out of test_helper.rb
+    Template.delete_all
```

Comment:
I don't get why we delete_all here, can you explain the purpose to me?

Reply:
We're using the seeds.rb file to populate the test db but its not always clear how those records are setup (e.g. how many templates are there, which ones are published, who do they belong to, etc.) Its cleaner to just add what you need in the `setup` for each test that way you can guarantee that a change to seed.rb doesn't break your tests